tools/amp-validation/index.js

```diff
@@ -26,10 +24,12 @@ function checkEndpoints(devChannel) {
 }
 
 function runValidator(validator, devChannel) {
+    const isDev = process.argv[2] || false;
```

Comment:
the convention is to use `NODE_ENV=dev` and `process.env`, I'd prefix the command with the env var